docs/UsersGuide/source/ConfigWorkflow.rst

```diff
@@ -136,109 +181,427 @@ These variables apply only when using NCO mode (i.e. when ``RUN_ENVIR`` is set t
 
 CCPP Parameter
 ==============
-``CCPP_PHYS_SUITE``: (Default: "FV3_GFS_v15p2")
-   The CCPP (Common Community Physics Package) physics suite to use for the forecast(s).  The choice of physics suite determines the forecast model's namelist file, the diagnostics table file, the field table file, and the XML physics suite definition file that are staged in the experiment directory or the cycle directories under it.  Current supported settings for this parameter are “FV3_GFS_v15p2” and “FV3_RRFS_v1alpha”.
+``CCPP_PHYS_SUITE``: (Default: "FV3_GFS_v16")
+   This parameter indicates which :term:`CCPP` (Common Community Physics Package) physics suite to use for the forecast(s). The choice of physics suite determines the forecast model's namelist file, the diagnostics table file, the field table file, and the XML physics suite definition file, which are staged in the experiment directory or the :term:`cycle` directories under it. 
+   
+   **Current supported settings for this parameter are:** 
+
+   | "FV3_GFS_v16" 
+   | "FV3_RRFS_v1beta" 
+   | "FV3_HRRR"
+   | "FV3_WoFS"
+
+   **Other valid values include:**
+
+   | "FV3_GFS_2017_gfdlmp"
+   | "FV3_GFS_2017_gfdlmp_regional"
+   | "FV3_GFS_v15p2"
+   | "FV3_GFS_v15_thompson_mynn_lam3km"
+   | "FV3_RRFS_v1alpha"
+
+..
+   COMMENT: "FV3_WoFS" technically has not been merged yet... and is called NSSL? What should I put for now? Current Default is "FV3_GFS_v15p2" - need to make sure we change that. 
+
+Stochastic Physics Parameters
+================================
+
+For the most updated and detailed documentation of these parameters, see the `UFS Stochastic Physics Documentation <https://stochastic-physics.readthedocs.io/en/latest/namelist_options.html>`__.
+
+
+``NEW_LSCALE``: (Default: "TRUE") 
+   Use correct formula for converting a spatial legnthscale into spectral space. 
+
+Specific Humidity (SHUM) Perturbation Parameters
+---------------------------------------------------
+
+``DO_SHUM``: (Default: "FALSE")
+   Flag to turn Specific Humidity (SHUM) perturbations on or off. SHUM perturbations multiply the low-level specific humidity by a small random number at each time-step. The SHUM scheme attempts to address missing physics phenomena (e.g., cold pools, gust fronts) most active in convective regions. 
+
+``ISEED_SHUM``: (Default: "2")
+   Seed for setting the SHUM random number sequence.
+
+``SHUM_MAG``: (Default: "0.006") 
+   Amplitudes of random patterns. Corresponds to the variable ``shum`` in ``input.nml``.
+
+``SHUM_LSCALE``: (Default: "150000")
+   Decorrelation spatial scale in meters.
+
+``SHUM_TSCALE``: (Default: "21600")
+   Decorrelation timescale in seconds. Corresponds to the variable ``shum_tau`` in ``input.nml``.
+
+``SHUM_INT``: (Default: "3600")
+   Interval in seconds to update random pattern (optional). Perturbations still get applied at every time-step. Corresponds to the variable ``shumint`` in ``input.nml``.
+
+.. _SPPT:
+
```

Comment:
Should "_SPPT" exist above?  I don't see one for "SHUM" previously.  The SHUM description is all listed under "DO_SHUM".

Reply:
Yes. It's a cross-referencing mechanism. Since I didn't cross-reference SHUM, I didn't add one for SHUM. The ".. _SPPT:" label doesn't show up visibly in the docs on RTD, but RTD uses it to reference that portion of the text, and you'll see the cross-reference if you're reading the raw code (just like with comments).